src/js/tracing/routingInstrumentation.ts

```diff
@@ -40,6 +44,8 @@ export interface RoutingInstrumentationInstance {
 export class RoutingInstrumentation implements RoutingInstrumentationInstance {
   public static instrumentationName: string = 'base-routing-instrumentation';
 
+  public readonly name: string = RoutingInstrumentation.instrumentationName;
```

Comment:
L: Should we kill `instrumentationName` then and use `name` only?

Reply:
We are actually using the static value outside of the class.
https://github.com/getsentry/sentry-react-native/blob/77a228f62596771bde0e7ddc3b01257a863cab7a/src/js/tracing/reactnavigationv4.ts#L370
So I would like to keep it, as we have id and name in integrations.